client/signup/config/test/index.js

```diff
@@ -8,10 +8,22 @@ import isEmpty from 'lodash/isEmpty';
 /**
  * Internal dependencies
  */
-import flows from '../flows';
-import steps from '../steps';
+import useFilesystemMocks from 'test/helpers/use-filesystem-mocks';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'index', () => {
+	let flows, steps;
+
+	useFilesystemMocks( __dirname );
```

Comment:
I forgot that filesystem mocks uses also mockery behind the scenes... Nice catch!